scripts/meta/west/commands/run_common.py

```diff
@@ -94,6 +94,9 @@ def add_parser_common(parser_adder, command):
         '--openocd-search',
         help='Path to add to OpenOCD search path, if applicable')
 
+    group.add_argument('--jlink',
```

Comment:
Please add any jlink-specific items to the JLink-specific runner. The openocd stuff in here is all legacy.

Also west is developed separately: https://github.com/zephyrproject-rtos/west. This is just a copy of west. Please don't submit patches to west here; they won't be accepted.

Reply:
As an update, the runners ended up going into `scripts/west_commands/runner` in the final `topic-west` branch that is likely to be merged.